modules/tinymce/src/core/main/ts/caret/FakeCaret.ts

```diff
@@ -70,12 +71,12 @@ const getAbsoluteClientRect = (root: HTMLElement, element: HTMLElement, before:
   return clientRect;
 };
 
-const trimInlineCaretContainers = (root: Node): void => {
+const trimInlineCaretContainers = (root: HTMLElement): void => {
   let contentEditableFalseNodes, node, sibling, i, data;
 
-  contentEditableFalseNodes = DomQuery('*[contentEditable=false]', root);
+  contentEditableFalseNodes = SelectorFilter.descendants(SugarElement.fromDom(root), '*[contentEditable=false]');
```

Comment:
It might be worth doing a general audit at some point - any selector that doesn't rely on sizzle-specific extensions will definitely be faster using `querySelectorAll`.

Reply:
Agreed :+1: